user_documentation/configuration.tex

```diff
@@ -1,11 +1,22 @@
 \section{External Macro Libraries and COPY Members}
 \label{sec:configuration}
 
-The HLASM Language Support extension looks for locally stored members when a macro or COPY instruction is evaluated. The paths of these members are specified in two configuration files in the \TT{.hlasmplugin} folder of the currently open workspace. Ensure that you configure these files before using macros from separate files or the COPY instruction.
+The HLASM Language Support extension looks for locally stored members when a macro or COPY instruction is evaluated. The paths of these members are specified in two configuration files in the \TT{.hlasmplugin} folder of the currently open workspace:
+\begin{itemize}
+	\item  \TT{proc\_grps.json} defines \textbf{processor groups} by assigning a group name to a list of directories. Hence, the group name serves as an unique identifier of a HLASM library set defined by a list of directories.
```

Comment:
```suggestion
	\item  \TT{proc\_grps.json} defines \textbf{processor groups} by assigning a group name to a list of directories. Hence, the group name serves as a unique identifier of a HLASM library set defined by a list of directories.
```

Reply:
HLASM library set? Ako mnozina HLASM kniznic? Ak hej, otocil by som to - 'a set of HLASM libraries'.

Reply:
jop